webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/__snapshots__/AutoComplete.test.js.snap

```diff
@@ -3,6 +3,7 @@
 exports[`AutoComplete rendering renders AutoComplete 1`] = `
 <div>
   <OnClickOutside(TypeaheadContainer(WrappedTypeahead))
+    controller="models"
```

Comment:
looks like a hardcoded path somewhere?

Reply:
yes, added optional props in the Autocomplete `{...this.props}` and now it is being shown in the snapshot.